recipes/xsimd/run_test.bat

```diff
@@ -0,0 +1,7 @@
+conda env create -f .\test\test-environment.yml
```

Comment:
Could you please share the relevant log snippet and link to the log? Would be great to figure out what is going on here.